app/services/storage_root_migration_service.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+# Migrates Complete Moab records to a new Moab Storate Root.
+class StorageRootMigrationService
```

Comment:
Should we indicate that this code is temporary and expected to go away?

Reply:
i think maybe an issue or (slack/outlook) reminder or something to nudge us to revisit would make sense, but there doesn't seem to be a clear deprecation horizon for this.  marking resolved, but feel free to unresolve if that seems wrong.